src/sentry/utils/audit.py

```diff
@@ -35,3 +48,63 @@ def create_audit_entry(request, transaction_id=None, logger=None, **kwargs):
         logger.info(entry.get_event_display(), extra=extra)
 
     return entry
+
+
+def create_org_delete_log(entry):
+    """ Creates a log of deleted organization """
+    delete_log = DeletedOrganization()
+    organization = Organization.objects.get(id=entry.target_object)
+
+    delete_log.name = organization.name
+    delete_log.slug = organization.slug
+    delete_log.date_created = organization.date_added
+
+    complete_delete_log(delete_log, entry)
+
+
+def create_project_delete_log(entry):
+    """ Creates a log of deleted project """
+    delete_log = DeletedProject()
+
+    project = Project.objects.select_related('team').get(id=entry.target_object)
+    delete_log.name = project.name
+    delete_log.slug = project.slug
+    delete_log.date_created = project.date_added
+
+    delete_log.organization_id = entry.organization.id
+    delete_log.organization_name = entry.organization.name
+    delete_log.organization_slug = entry.organization.slug
+
+    team = project.team
+    delete_log.team_id = team.id
+    delete_log.team_name = team.name
+    delete_log.team_slug = team.slug
+
+    complete_delete_log(delete_log, entry)
+
+
+def create_team_delete_log(entry):
+    """ Creates a log of the deleted team """
+    delete_log = DeletedTeam()
+
+    team = Team.objects.get(id=entry.target_object)
+    delete_log.name = team.name
+    delete_log.slug = team.slug
+    delete_log.date_created = team.date_added
+
+    delete_log.organization_id = entry.organization.id
+    delete_log.organization_name = entry.organization.name
+    delete_log.organization_slug = entry.organization.slug
+
+    complete_delete_log(delete_log, entry)
+
+
+def complete_delete_log(delete_log, entry):
+    """ Adds common information on a delete log from an audit entry
```

Comment:
Now this comment is slightly more useful. :)

One note on formatting:

```python
def foo():
    """
    Adds common information on a delete log...
    """
```

Is a bit cleaner and more along with what we do elsewhere.

Reply:
I think in the past I'd been told to do 
``` python
def docstr_example():
     """ Comment like this when there is one line """


def docstr_example_2():
     """ 
     Comment like this
     When there are multiple lines
     """"
````

I don't have any real preference. I'll do the latter regardless of line count from now on.